packages/contracts-bedrock/slither.config.json

```diff
@@ -1,9 +1,11 @@
 {
-  "detectors_to_exclude": "assembly-usage,block-timestamp,naming-convention,solc-version,low-level-calls,boolean-equality",
+  "detectors_to_exclude": "incorrect-shift-in-assembly",
+  "fail_high": true,
+  "fail_pedantic": false,
+  "exclude_optimization": true,
   "exclude_informational": true,
   "exclude_low": true,
   "exclude_medium": true,
```

Comment:
> Print Medium severity findings but not fail on them.
Medium detectors are being excluded

Reply:
I think I changed my mind about printing without failing, in order to avoid confusion and reduce the noise level.